src/ORM/Table.php

```diff
@@ -1853,7 +1853,7 @@ protected function _onSaveSuccess($entity, $options)
             $this,
             $entity,
             $options['associated'],
-            ['_primary' => false] + $options->getArrayCopy()
+            ['_primary' => false] + ['parent' => $entity] + $options->getArrayCopy()
```

Comment:
It would be better to prefix the special key `parent` with underscore, like it's done for `_primary`.